editors/code/package.json

```diff
@@ -1089,7 +1089,7 @@
                 },
                 "rust-analyzer.hover.memoryLayout.enable": {
                     "markdownDescription": "Whether to show memory layout data on hover.",
-                    "default": true,
+                    "default": false,
```

Comment:
instead of changing this here, you're going to `crates/rust-analyzer/src/config.rs` and change this line: https://github.com/rust-lang/rust-analyzer/blob/a2e274142f35d21fd28d28655f4af8e8531ab649/crates/rust-analyzer/src/config.rs#L373. You'll then want to run `cargo test`, as that will run a test that updates a `package.json` (and a few other files, iirc) to the appropriate default.

Put differently, this isn't the source of truth: `config.rs` is.

Reply:
Thanks, fixed.